contracts/Paper.sol

```diff
@@ -0,0 +1,156 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.2;
+
+import "@openzeppelin/contracts@4.3.0/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts@4.3.0/token/ERC20/extensions/ERC20Snapshot.sol";
+import "@openzeppelin/contracts@4.3.0/token/ERC721/extensions/IERC721Enumerable.sol";
+import "@openzeppelin/contracts@4.3.0/access/Ownable.sol";
+
+contract Paper is ERC20, ERC20Snapshot, Ownable {
+    // Dope Wars Loot contract is available at https://etherscan.io/address/0x8707276DF042E89669d69A177d3DA7dC78bd8723
+    IERC721Enumerable public dopeWarsLootContract = IERC721Enumerable(0x8707276DF042E89669d69A177d3DA7dC78bd8723);
+
+    // 8000 tokens number 1-8000
+    uint256 public tokenIdStart = 1;
+    uint256 public tokenIdEnd = 8000;
+
+    // Give out 50% of tokens, evenly split across each NFT
+    uint256 public supply = 1000000000 * (10**decimals());
+    uint256 public paperPerTokenId = supply / 2 / tokenIdEnd ;
```

Comment:
I thought we were going to airdrop only 25% of the supply, not half of it.

Reply:
```suggestion
    uint256 public paperPerTokenId = supply / 4 / tokenIdEnd ;
```

Reply:
we're gonna allow the dao to mint tokens rather than airdrop to it